opslevel/resource_opslevel_trigger_definition.go

```diff
@@ -200,6 +215,11 @@ func resourceTriggerDefinitionUpdate(d *schema.ResourceData, client *opslevel.Cl
 		input.ResponseTemplate = &responseTemplate
 	}
 
+	if d.HasChange("entity_type") {
```

Comment:
I'm 99% this is the logic that will need to be different because of the default in the API.  I'm going to pull this change down later today and test it manually but I think we are going to need handle the default here as the library opslevel-go won't "unset" the value during update.

Reply:
this was correct.  I made a commit that fixes this.